hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/fees/ConsensusServiceSimpleFeesSuite.java

```diff
@@ -0,0 +1,239 @@
+import org.junit.jupiter.api.DynamicTest;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Tag;
+
+@Tag(SIMPLE_FEES)
+@Tag(MATS)
+@HapiTestLifecycle
+public class ConsensusServiceSimpleFeesSuite {
+
+    @FunctionalInterface
+    public interface OpsProvider {
+        List<SpecOperation> provide();
+    }
+
+    private Stream<DynamicTest> compare(OpsProvider provider) {
+        List<SpecOperation> opsList = new ArrayList<>();
+        opsList.add(overriding("fees.simpleFeesEnabled", "false"));
+        opsList.add(withOpContext((spec, op) -> {
+            System.out.println("old fees");
+        }));
+        opsList.addAll(provider.provide());
+        opsList.add(overriding("fees.simpleFeesEnabled", "true"));
+        opsList.add(withOpContext((spec, op) -> {
+            System.out.println("new fees");
+        }));
+        opsList.addAll(provider.provide());
+        return hapiTest(opsList.toArray(new SpecOperation[opsList.size()]));
+    }
+
+    @Nested
+    class TopicFeesComparison {
+        private static final String PAYER = "payer";
+        private static final String ADMIN = "admin";
+
+        @HapiTest()
+        @DisplayName("compare create topic")
+        final Stream<DynamicTest> createTopicPlainComparison() {
+            return compare(() -> Arrays.asList(
+                    cryptoCreate(PAYER).balance(ONE_HUNDRED_HBARS),
+                    createTopic("testTopic")
+                            .blankMemo()
+                            .payingWith(PAYER)
+                            .fee(ONE_HBAR)
+                            .via("create-topic-txn"),
+                    validateChargedUsd("create-topic-txn", 0.01)));
+        }
+
+        @HapiTest()
+        @DisplayName("compare create topic with admin key")
+        final Stream<DynamicTest> createTopicWithAdminComparison() {
+            return compare(() -> Arrays.asList(
+                    newKeyNamed(ADMIN),
+                    cryptoCreate(PAYER).balance(ONE_HUNDRED_HBARS),
+                    createTopic("testTopic")
+                            .blankMemo()
+                            .payingWith(PAYER)
+                            .adminKeyName(ADMIN)
+                            .fee(ONE_HBAR)
+                            .via("create-topic-admin-txn"),
+                    // TODO: adjust this once we get final pricing
+                    validateChargedUsdWithin("create-topic-admin-txn", 0.0163, 30)));
+        }
+
+        @HapiTest()
+        @DisplayName("compare create topic with payer as admin key")
+        final Stream<DynamicTest> createTopicWithPayerAdminComparison() {
+            return compare(() -> Arrays.asList(
+                    newKeyNamed(ADMIN),
+                    cryptoCreate(PAYER).balance(ONE_HUNDRED_HBARS),
+                    createTopic("testTopic")
+                            .blankMemo()
+                            .payingWith(PAYER)
+                            .adminKeyName(PAYER)
```

Comment:
Since the base fee doesn't have adminKey, we can add another test for validating baseFee without adminKey. I think with current implementation we should get 0.01.